config/jobs/kubernetes-sigs/lws/OWNERS

```diff
@@ -2,4 +2,7 @@
 
 approvers:
 - ahg-g
+- kerthcet
```

Comment:
Can you please add @Edwinhr716 as well?

btw, @Edwinhr716 where did you add yourself to get maintainer access to the repo? I forgot where we do that.  @kerthcet should be added there as well

Reply:
done.

Reply:
ping @ahg-g